content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,150 @@
+working properly with the :abbr:`IoT (Internet of Things)` box.
+
+For both the Windows :abbr:`IoT (Internet of Things)` (Odoo 16 and higher) and physical :abbr:`IoT
+(Internet of Things)` box, this process can be performed manually from the :abbr:`IoT (Internet of
+Things)` box homepage. Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to
+:menuselection:`IoT app --> IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT
+(Internet of Things)` box.
+
+Next, click :guilabel:`Handlers list` and then select :guilabel:`Load Handlers` at the bottom of the
+page.
+
+.. image:: updating_iot/load-handlers.png
+   :align: center
+   :alt: Handlers list on an IoT box with the load handlers button highlighted.
+
+.. important::
+   Handlers code is fetched from the configured server and it needs to be up-to-date to have the
+   latest fixes and patches. See :ref:`upgrade/homepage_upgrade`.
+
+.. note::
+   The handlers update is also performed automatically each time the :abbr:`IoT (Internet of
+   Things)` box is restarted. The only exception to this process is if the :guilabel:`Automatic
+   drivers update` is unchecked in the form view of the :abbr:`IoT (Internet of Things)` box on the
+   Odoo server. This setting can be reached by going to :menuselection:`IoT App --> Select the IoT
+   box --> Automatic drivers update`.
+
+.. _upgrade/homepage_upgrade:
+
+Update from the IoT box home page
+=================================
+
+.. important::
+   This update does not apply to the Windows :abbr:`IoT (Internet of Things)` box (Odoo 16 and
+   higher). To update the Windows :abbr:`IoT (Internet of Things)`, first, uninstall the previous
+   version of the Odoo Windows program and then reinstall it, with the most up-to-date installation
+   package. To begin the installation, navigate to the Odoo 16 or higher installation package for
+   Enterprise or Community - Windows edition at `Odoo’s download page <https://odoo.com/download>`_.
+
+In the background, the :abbr:`IoT (Internet of Things)` box uses a version of Odoo code to run and
+connect to the Odoo database. This code may need to be updated in order for the :abbr:`IoT (Internet
+of Things)` box to operate effectively. This operation should be completed on a routine basis, to
+ensure up-to-date :abbr:`IoT (Internet of Things)` processes and core system.
+
+Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to :menuselection:`IoT app -->
+IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT (Internet of Things)` box.
+Then click on :guilabel:`Update` (next to the version number).
+
+If a new version of the :abbr:`IoT (Internet of Things)` Box image is available, an
+:guilabel:`Upgrade to _xx.xx_` button will appear at the bottom of the page. Click this button to
+upgrade the unit and the :abbr:`IoT (Internet of Things)` box will then flash itself to the new
+version. All of the previous configurations will be saved.
```

Comment:
it's a bit confusing here, note that upgrading the image will also update the core code, but not necessarily to the latest version (but to the version of the code on which the image was built on which is in consequence newer for newer images).
I find a bit confusing to highlight this button in the main section of the page as it is in fact the "Upgrade" button that interest us for this update

Reply:
I understand that the latest image is the go-to for all versions now, thus a newer version if that file was updated.